.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Fixes #
```

Comment:
Sounds like this is only for bugs, is it?

Reply:
this is for linking issue with the pull request.

eg. Fixes  issue_link

once the pr is merged , issue will automatically closed if its linked with the pr

Reply:
![image](https://user-images.githubusercontent.com/55848322/155869479-18de560a-ebba-4fe3-bb40-8a6b97fc3edb.png)
![image](https://user-images.githubusercontent.com/55848322/155869484-6f97087c-28cc-413f-8f9c-97ca4253f9a2.png)

Reply:
Got it, thanks!  

How about adding [some points of this example](https://github.com/conventional-changelog/commitlint/blob/master/.github/PULL_REQUEST_TEMPLATE.md)?

I.e.:

- Motivation and Context
- Types of changes
- Checklist:
  -  I have updated the documentation accordingly
  - I have added tests to cover my changes

Reply:
adding the type of change is a good idea! 

Motivation and Context, and Checklist. both can be handled by ```describe the changes you have made in this PR```
what do you think about this? 

or we can add them as a subsection of  ``````describe the changes you have made in this PR``` section

Reply:
I was avoiding defining their cause, except for major PR, not every regular pr needs a checklist to be defined. also context and motivation might not be needed when the user describes about the changes made by pr.

Reply:
# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

General boilerplate can be this would you like any specific tests here to be included?

Reply:
This looks good. Let's go with this for now.

Reply:
Done required changes, additionally added checklist to ensure basic things are done or not.

Reply:
checklist for PR, representing work done till now, can be created by the contributor itself in ```describe the changes``` section if required